plugin/trino-mongodb/src/test/java/io/trino/plugin/mongodb/TestMongoConnectorTest.java

```diff
@@ -810,6 +809,29 @@ protected void verifyTableNameLengthFailurePermissible(Throwable e)
         assertThat(e).hasMessageMatching(".*fully qualified namespace .* is too long.*|Qualified identifier name must be shorter than or equal to '120'.*");
     }
 
+    @Override
+    protected void verifySetColumnTypeFailurePermissible(Throwable e)
+    {
+        assertThat(e).hasMessageContaining("Cannot change type");
+    }
+
+    @Override
+    protected Optional<SetColumnTypeSetup> filterSetColumnTypesDataProvider(SetColumnTypeSetup setup)
```

Comment:
Do we additional tests for structural data types like array ?

Reply:
Specific to `rowType` where we could add a new field, drop a new field and rename a field ?

Reply:
BaseConnectorTest already covers array and row types in `io.trino.testing.BaseConnectorTest#setColumnTypeSetupData`

Reply:
Oops I missed it. Thanks for pointing it out.